Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -29,8 +58,19 @@ public static void Reverse(int[] array)
         /// <returns>A new array with the last element removed. If an array is empty or null, returns input array.</returns>
         public static int[] RemoveLast(int[] array)
```

Comment:
When you have RemoveAt, can't you reuse it when implementing RemoveFirst/RemoveAt?